support/iconsClassesGenerator.js

```diff
@@ -102,30 +102,53 @@ for (const [cKEditorIcon, clayIcon] of Object.entries(iconsConfig.icons)) {
 	let directionClass = '';
 	let svgDataList = [];
 
-	if (typeof clayIcon === 'string') {
-		const svgData = fs.readFileSync(
-			`${sourceIconsPath}/${clayIcon}.svg`,
-			'utf8'
-		);
+	let svgData;
```

Comment:
Why extracting this?

Reply:
Why have 3 `const` when you can have one `let`

Reply:
Since you're going this way... why have `let` at all? just inline the calls, you're not using `svgData` for anything else...

Looking on a quick look, I think you could even do something like:

```javascript
const direction = typeof clayIcon === 'object' || clayIcon.rtl ? 'rtl' : 'ltr';
const iconPath = clayIcon === 'string' ? clayIcon : clayIcon.ltr | clayIcon.rtl;

iconCssContent += getCss(
    fs.readFileSync(`${sourceIconsPath/${iconPath}}.svg`, 'utf8'),
    ckEditorIcon,
    direction
);
```

Not saying this reads better, but it's probably as hard to read as the one we have right now because all the code looks so similar that it's hard to tell the branches apart.

Reply:
The thing is that when `clayIcon` is an object, it usually has [both](https://github.com/liferay/liferay-ckeditor/blob/09632a29a40f92c9bf5088d166520873f81125e3/skins/moono-lexicon/icons/icons.json#L15-L30) `ltr` and `rtl` properties, and we want to generate the class for both so this in this loop

```js
const direction = typeof clayIcon === 'object' || clayIcon.rtl ? 'rtl' : 'ltr';
const iconPath = clayIcon === 'string' ? clayIcon : clayIcon.ltr | clayIcon.rtl;
```

won't do it.

One thing I'll can do is get rid of the [unused](https://github.com/liferay/liferay-ckeditor/blob/09632a29a40f92c9bf5088d166520873f81125e3/support/iconsClassesGenerator.js#L102-L103) variables in the loop

Reply:
You're totally right 👍

Reply:
But let's not touch this too much :)

Reply:
@jbalsas yes that would even be better. 
I agree, I'm going to focus on #122 right now, and when the time comes we can improve or clean up these helpers scripts.